executor/index_merge_reader_test.go

```diff
@@ -454,3 +454,91 @@ func (test *testSerialSuite2) TestIndexMergeReaderMemTracker(c *C) {
 	c.Assert(err, IsNil)
 	c.Assert(bytes, Greater, 0.0)
 }
+
+func (test *testSerialSuite2) TestIndexMergeWithCorrelatedColumns(c *C) {
```

Comment:
Should we add these into planner pkg?   see  planner/core/testdata

Reply:
done